charts/ccpay-callback-function/values.preview.template.yaml

```
@@ -5,19 +5,20 @@ function:
     APPINSIGHTS_INSTRUMENTATIONKEY: 124e39c3-6dad-49ef-aede-d2cddac865c2
     SERVICE_CALLBACK_SUBSCRIPTION: pr-subscription-name
   secrets:
-    SERVICE_CALLBACK_BUS_CONNECTION:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-servicecallbacktopic
+    SB_ACCESS_KEY:
+      secretRef: fees-pay-sb-preview
+      key: primaryKey
+    SB_CONNECTION_STRING:
+      secretRef: fees-pay-sb-preview
       key: connectionString
-    SERVICE_CALLBACK_SUBSCRIPTION:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-servicecallbacktopic
-      key: subscriptionName
 servicebus:
   resourceGroup: fees-pay-rg
+  sbNamespace: "fees-pay-servicebus-preview"
   teamName: "Fees Pay"
   location: ukwest
   serviceplan: standard
   setup:
     topics:
-      - name: servicecallbacktopic
-        subscriptionNeeded: yes
+    - name: servicecallbacktopic
```

> * NamespacesTopicsSubscription.servicebus.azure.com "ccpay-callback-function-pr-98-servicebus-servicecallbacktopic" is invalid: spec.azureName: Too long: may not be longer than 50

Not sure if you need to change anything else but maybe you could do:

```suggestion
    - name: callback
```